packages/dev/src/main.ts

```diff
@@ -184,6 +193,7 @@ export class NetlifyDev {
       static: options.staticFiles?.enabled !== false,
     }
     this.#functionsServePath = path.join(projectRoot, '.netlify', 'functions-serve')
+    this.#imageAllowedDomains = options.images?.remoteImages ?? []
```

Comment:
same here, maybe avoid this misleading terminology
```suggestion
    this.#imageAllowedRemoteUrlPatterns = options.images?.remoteImages ?? []
```